src/AutoRest.CSharp/MgmtTest/Extensions/CodeWriterExtensions.cs

```diff
@@ -142,7 +142,8 @@ private static CodeWriter AppendDataFactoryElementValue(this CodeWriter writer,
             }
             else
             {
-                CSharpType literlType = type.Arguments[0];
+                const int DFE_LITERAL_TYPE_ARG_INDEX = 0;
+                CSharpType literlType = type.Arguments[DFE_LITERAL_TYPE_ARG_INDEX];
```

Comment:
This is still a magic number. How about we just simply use `type.Arguments.First()` which has more clear semantic meaning.